autograder-api/src/main/java/de/firemage/autograder/api/AbstractLinter.java

```diff
@@ -0,0 +1,71 @@
+package de.firemage.autograder.api;
+
+import fluent.bundle.FluentBundle;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Locale;
+import java.util.function.Consumer;
+
+public interface AbstractLinter {
+    List<? extends AbstractProblem> checkFile(Path file, JavaVersion version, CheckConfiguration checkConfiguration, Consumer<Translatable> statusConsumer) throws LinterException, IOException;
+    String translateMessage(Translatable translatable);
+    FluentBundle getFluentBundle();
```

Comment:
Same problem here, why do we expose the internals? Isn't `translateMessage` enough? If we want a method for explicitly specifying the language, we could have this instead?:
```java
default String translateMessage(Translatable translatable) {
    return this.transalteMessage(translatable, this.locale);
}

String translateMessage(Translatable translatable, Locale locale);
```

Reply:
You mean getFluentBundle(), right? I guess it can be removed; it was only needed at some point while implementing the PR

Reply:
Yes